### DISCUSSION TITLE
Error installing .NET templates

### DISCUSSION THREAD

**Discussion by rbennett00** [learn-monogame/learn-monogame.github.io#36] [General]
As I'm going through the steps in [Get Started](https://learn-monogame.github.io/how-to/get-started/), I encountered an error trying to install the templates: 

```
Error: Failed to load template from C:\Users\rbenn\.templateengine\packages\MonoGame.Templates.CSharp.3.8.1.303.nupkg(/content/content/MonoGame.Application.UWP.CoreApp.CSharp/.template.config/template.json).
Details: 'identity' is missing or is an empty string.
```
It seems that others are [having the same issue](https://community.monogame.net/t/vs-2022-templates/17905).

If this error is non-fatal and can be ignored, then I recommend noting it in the documentation. If it is a blocking error then recommend including remediation steps (if known).

**Comment by Apostolique** [comment]
Added a note. It's safe to ignore.

Eventually that package will not be included with MonoGame anymore. https://github.com/MonoGame/MonoGame/issues/8113#issuecomment-1870743694
